src/xrpld/app/tx/detail/NFTokenModify.cpp

```diff
@@ -37,6 +37,10 @@ NFTokenModify::preflight(PreflightContext const& ctx)
     if (NotTEC const ret = preflight1(ctx); !isTesSuccess(ret))
         return ret;
 
+    auto const flags = ctx.tx.getFlags();
+    if (flags & tfNFTokenModifyMask)
+        return temINVALID_FLAG;
```

Comment:
```suggestion
    if (ctx.tx.getFlags() & tfNFTokenModifyMask)
        return temINVALID_FLAG;
```

Don't need to create a new variable if it's used only once

Reply:
fixed